src/Languages/Properties/MathProperties.ts

```diff
@@ -8,12 +8,14 @@ export class MathProperties {
      * How to retrieve the absolute value of a number.
      */
     public absolute: NativeCallProperties;
-
+    /**
+     * How to retrieve the largest previous integer of a number.
+     */
+    public floor: NativeCallProperties;
     /**
      * Any imports native math commands require.
      */
     public requiredImports: { [i: string]: string[] };
-
```

Comment:
Why remove these lines? Please keep them, and add blank lines around `public floor`.